src/sandbox/settings/form.js

```diff
@@ -81,31 +81,55 @@ export const createSettings = memoize((platform) => {
           ],
         },
         {
-          key: CONFIG.KEY,
-          name: CONFIG.DROP_OFF_DAYS,
-          component: CCheckboxGroup,
-          props: {
-            // Map the weekdays to options. If the day is sunday set index to 0.
-            options: weekdays.map((day, index) => ({
-              value: index === weekdays.length - 1 ? 0 : index + 1,
-              text: day,
-            })),
-          },
+          title: CONFIG.DROP_OFF_DAYS,
+          settings: [
+            ...perCarrier({
+              key: CONFIG.KEY,
+              name: CONFIG.DROP_OFF_DAYS,
+              component: CCheckboxGroup,
+              props: {
+                // Map the weekdays to options. If the day is sunday set index to 0.
+                options: weekdays.map((day, index) => ({
```

Comment:
Deze line is niet gecovered, dat was al zo, maaaaaar, codecov vindt het niet goed. Zou je kunnen kijken of je dit er makkelijk in kunt krijgen? Er is ergens vgm een sandbox components test, misschien kan het daarin ofzo